include/TrustWalletCore/TWAnyAddress.h

```diff
@@ -35,9 +35,6 @@ struct TWAnyAddress* _Nullable TWAnyAddressCreateWithString(TWString* _Nonnull s
 TW_EXPORT_STATIC_METHOD
 struct TWAnyAddress* _Nonnull TWAnyAddressCreateWithPublicKey(struct TWPublicKey* _Nonnull publicKey, enum TWCoinType coin);
 
-TW_EXPORT_METHOD
-void TWAnyAddressDelete(struct TWAnyAddress* _Nonnull address);
```

Comment:
I think this will cause memory leak for iOS / Android

Reply:
I don't think it will, see my comment